cves/CVE-2017-14056.yml

```diff
@@ -236,8 +265,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: Demuxers are a part of the avformat subsystem.
+  name: avformat
```

Comment:
Perhaps give more detail about what the avformat subsystem is and what it is responsible for.